cmd/datasetArchiver/main.go

```diff
@@ -14,15 +14,24 @@ import (
 	"log"
 	"net/http"
 	"os"
-	"github.com/paulscherrerinstitute/scicat/datasetUtils"
 	"strings"
 	"time"
 
+	"github.com/paulscherrerinstitute/scicat/datasetUtils"
+
```

Comment:
extra line. Is this on purpose? then should there be one before as well (I am guessing some sort of PEP8 for go?)

Reply:
it's the golang extension for vscodium/vscode. It autoformats everything everytime I hit Ctrl + S, so I think this is the official suggestion for formatting (and go formatting can be extremely specific, like not allowing for non-inline block starts).

Reply:
ah ok, thanks. Should we keep it for the last commit as well then?